/*
  Device Vehicle Mapping

  This section holds messages used in self-mapping service.

  The version of the OpenAPI document: 2

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.devicevehiclemapping.v2;


import public "backend_to_device_message_meta.proto";

message UpdateMappingStatusCommand {

  BackendToDeviceMessageMeta meta = 3347973;
  MappingStatusEnum mappingStatus = 336559521;

}

enum MappingStatusEnum {
    VIN_RECEIVED = 0;
    MAPPED = 1;
    UNMAPPED = 2;
    FAILURE = 3;
  }